spring-resttemplate-2/src/main/java/com/baeldung/resttemplate/json/model/User.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.resttemplate.json.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class User {
+    private int id;
+    private String name;
+    private List<Address> addressList;
+
+    public User(int id, String name, List<Address> addressList) {
```

Comment:
This POJO is more complex than it needs to be for the example. We're just looking for a situation where there's a list of objects coming back from an endpoint. This object has a list of other things in it. That seems off topic.

Reply:
I think the ticket or the stackoverflow link mentioned processing nested objects - I think it is a good example to see

Reply:
Ok.